test/integration/framework/etcd.go

```diff
@@ -198,7 +198,7 @@ func EtcdMain(tests func() int) {
 		// like k8s.io/klog/v2.(*loggingT).flushDaemon()
 		// TODO(#108483): Reduce this number once we address the
```

Comment:
ToDo can be deleted.

Reply:
hopefully can be deleted soon, but there are still some other leaks https://github.com/kubernetes/kubernetes/issues/108483#issuecomment-1195714805